### ISSUE TITLE
Why can't I download when I open the chrome debugging tool?

### ISSUE THREAD

**Issue by wang1212** [jimmywarting/StreamSaver.js#136]
- environment
Chrome Version: 80.0.3970.5 (dev)

---
Sorry, this may be an obvious problem, but I don't quite understand.

I used your library in my project, but when I open Chrome Developer Tools, the console will get a 404 exception and the file cannot be downloaded. Then, I tried to copy your sample code locally, and opened a local server, and found that the same problem occurs. Fortunately, I accidentally closed the developer tools in Chrome and found that the problem disappeared and everything worked fine in my project.

**Comment by jimmywarting** [comment]
hmm, weird...

well, the streamsaver.js is heavily based on service workers to accomplish the streaming download (emulating what a server dose to save files and spitting out chunks of data)

so my best guess would be that you have some kind of "update on reload" or "bypass for network" or something

<img width="670" alt="Skärmavbild 2019-12-18 kl  11 24 35" src="https://user-images.githubusercontent.com/1148376/71078364-8abb7a80-2189-11ea-8c5d-f2b942cbbaf1.png">

**Comment by jimmywarting** [comment]
eventually i would like to be able to use WICG/native-filesystem api #73 to avoid service workers

i have experimented with it in canary but it doesn't feel ready for implementing yet...

this is also an issue that i would like to be solved before even thinking of implementing it https://github.com/WICG/native-file-system/issues/80

**Comment by wang1212** [comment]
First of all, thank you very much for your reply.

I did use Google [Workbox's webpack plugin](https://developers.google.cn/web/tools/workbox/) to implement the PWA application, so I turned on the **update on reload** and **bypass for network** options in Chrome's developer tools. 

However, after restarting my computer this morning, I found that the problem has disappeared, and it seems to be caused by other unknown reasons.

Finally, thank you for the library you provided to help me solve the problem of file download.
